packages/docusaurus-1.x/lib/server/blog.js

```diff
@@ -31,9 +28,7 @@ function fileToUrl(file) {
   }
   return path
     .basename(file)
-    .replace('-', '/')
-    .replace('-', '/')
-    .replace('-', '/')
+    .replace(/-/g, '/')
```

Comment:
This replaces ALL the hyphens with / but we only want to do that for the first 3 - because they are dates. Try it out and see for yourself.

Reply:
Thank you